ImplementationGuide/markdown/Datenobjekte/ISiKTerminblockSlot.md

```diff
@@ -74,29 +74,25 @@ Für die Ressource Slot MUSS die REST-Interaktion "READ" implementiert werden.
 
     ```GET [base]/Slot?schedule=Schedule/ISiKKalenderExample```
 
-    Anwendungshinweise: Weitere Informationen zur Suche nach "Slot.schedule" finden sich in der [FHIR-Basisspezifikation - Abschnitt "reference"](https://hl7.org/fhir/R4/search.html#reference).
+    Anwendungshinweise: In diesem Fall ist auch ein Chaining auf weitere verknüpfte Akteure möglich: `GET https://example.org/fhir/Slot?schedule.actor:HealthcareService.type=https://example.org/fhir/CodeSystem/Behandlungsleistung|CT`
+    Weitere Informationen zur Suche nach "Slot.schedule" finden sich in der [FHIR-Basisspezifikation - Abschnitt "reference"](https://hl7.org/fhir/R4/search.html#reference).
 
-3. Der Suchparameter "status" MUSS unterstützt werden:
+3. Der Suchparameter "status" MUSS in Kombination mit einer Chaining-Abfrage auf Schedule unterstützt werden:
 
     Beispiele:
 
-    ```GET [base]/Slot?status=busy```
+    ```GET [base]/Slot?status=free&schedule.actor:Practitioner.name=Musterarzt```
 
-    Anwendungshinweise: Weitere Informationen zur Suche nach "Slot.status" finden sich in der [FHIR-Basisspezifikation - Abschnitt "Token Search"](https://hl7.org/fhir/R4/search.html#token).
+    Anwendungshinweise: Der Suchparameter MUSS NICHT alleinstehend unterstützt werden. Weitere Informationen zur Suche nach "Slot.status" finden sich in der [FHIR-Basisspezifikation - Abschnitt "Token Search"](https://hl7.org/fhir/R4/search.html#token).
 
-4. Der Suchparameter "start" MUSS unterstützt werden:
+4. Der Suchparameter "start" MUSS in Kombination mit einer Chaining-Abfrage auf Schedule unterstützt werden:
```

Comment:
Hallo @f-peverali ,
das ist nicht das, was wir besprochen haben. Der Parameter start MUSS zusammen mit dem Parameter *schedule* verwendet werden und nicht *unbedingt* in Kombination mit einer Chaining-Abfrage. Chaining auf schedule-Parameter ist eine unabhängige Anforderung. 
@alexzautke

Reply:
betrifft auch den Parameter status

Reply:
siehe anderen PR: https://github.com/gematik/spec-ISiK-Terminplanung/pull/168